### PR TITLE
Create picoman.php

### PR DESCRIPTION
**Pull request by mamans86**
This is additional script for Picoman Custom Theme

PicoMan is a Mobile First Design TCExam Theme created by mamans86. For a while this theme is only applied to public area. If any spare time available i will create Admin Theme too. Maybe :)

This theme is built on top Pico.css (https://picocss.com/) 

## Features ##

1. Mobile First Design, Responsive, make TCExam more user friendly on small Device
2. Bootstrap like user interface
3. Password Field Toggler
4. Font Resizer Utility on Public Test Taker
5. Unsure Button to mark unsure question (using HTML5 LocalStorage to store unsure question index)
6. Maybe, it Work on any modern browser today :)

### CODE REVIEW

File: public/config.default/theme/picoman.php [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+	}
+}
+/*
+* Please don't remove or alter this credit, keep visible and reachable, contact me if you want to make donation. email: mamansulaeman86@gmail.com telegram: @mamans86 
+*/
+document.querySelector("body").insertAdjacentHTML("beforeend", "<div style='padding:0 1rem 1rem 1rem;text-align:center;background:var(--contrast);color:var(--background)'><span style='font-size:0.85rem'><a style='color:var(--color-warning)' href='https://mamans86.blogspot.com' target='blank'>PicoMan Theme</a> created with love by <a style='color:var(--color-warning)' href='https://mamans86.blogspot.com' target='blank'>mamans86</a></span></div>");
+document.getElementById("scrollayer").insertAdjacentHTML("beforeend", "<div style='position:fixed;bottom:1rem;right:1rem'><span style='font-size:0.75rem'><a href='https://mamans86.blogspot.com' target='blank'>TCExam PicoMan Theme</a></span></div>");
+/*
+* Please don't remove or alter this credit, keep visible and reachable, contact me if you want to make donation. email: mamansulaeman86@gmail.com telegram: @mamans86 
+*/
```
Comment (nicolaasuni) [inline public/config.default/theme/picoman.php:229]: Credits should go in:
https://github.com/tecnickcom/tcexam/blob/develop/admin/code/tce_page_info.php
Active links should point only to the official TCExam website.
Please refer and sign:
https://cla-assistant.io/tecnickcom/tcexam?pullRequest=350

File: public/config.default/theme/picoman.php [hunk 2]
```diff
@@ -0,0 +1,230 @@
+<script>
+/*!
+ * PicoMan v1.0 (https://mamans86.blogspot.com)
+ * email: mamansulaeman86@gmail.com telegram: @mamans86
+ * feel free to modify but please keep link to my blog always visible and reachable, and also please don't remove this comment block
+ * contact me if you want to make donation
+ * TCExam Mobile First Design Theme based on Pico.css (https://picocss.com), built with love by Maman Sulaeman - @mamans86
+ * eye & flag icon generated by icomoon.io
+ */
```
Comment (nicolaasuni) [inline public/config.default/theme/picoman.php:9]: It is OK to state the author but not additional licensing conditions.
Please refer to the TCExam license + https://cla-assistant.io/tecnickcom/tcexam?pullRequest=350

**Review by nicolaasuni — CHANGES_REQUESTED** [review]

**Review by mamans86 — COMMENTED** [review]
remove additional license, and active link

File: public/config.default/theme/picoman.php (hunk 1 above)
Comment (mamans86) [reply public/config.default/theme/picoman.php]: Btw, this script doesn't modify current links. The current link in bottom of page is still exist, visible and point to tcexam.org and tecnick.com

Cheers

**Review by nicolaasuni — APPROVED** [review]

**Pull request merged**